docs/administration-guide/sso.md

```diff
@@ -23,7 +23,6 @@ With paid versions, you have more options to help manage lots of people and grou
   - [Setting up SAML with Auth0][saml-auth0]
   - [Setting up SAML with Google][saml-google]
   - [Setting up SAML with Keycloak][saml-keycloak]
-  - [Documentation for other common IdPs][saml-other-idps]
```

Comment:
Could you also add links to:

- https://www.metabase.com/docs/latest/enterprise-guide/authenticating-with-saml-azure-ad.html
- https://www.metabase.com/docs/latest/enterprise-guide/saml-okta.html

I must have missed them when I last updated the SAML docs 😅